### PR TITLE
add travis CI

### LINKED ISSUE #10: Set up Travis
We'll need to use Docker to [use an Ubuntu with python3-systemd](http://packages.ubuntu.com/search?keywords=python3-systemd&searchon=names&suite=all&section=all) or archlinux or so.

### PR DESCRIPTION
**Pull request by RalfJung**
Fixes #10

### CODE REVIEW

**Comment by The-Compiler** [comment]
I just recently turned off Travis CI for the repo 😆 I turned it on again, and hope that'll cause Travis to pick it up. I suppose we need to use Docker here so we have a systemd available, right?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+services:
+  - docker
+env:
+  - PYTHON=python3.5
+  - PYTHON=python3.6
+
+install:
+  - docker build -t journalwatch/ci ci/"$PYTHON"
+script:
+  - docker run -v $(pwd):/build/ journalwatch/ci /usr/bin/"$PYTHON" /build/test_journalwatch.py
```
Comment (The-Compiler) [inline .travis.yml]: Why `$(pwd)` and not `$PWD`?

Comment (The-Compiler) [reply .travis.yml]: Also, you're running the Python file here, which does nothing. You'll need to run `"$PYTHON" -m pytest /build/test_journalwatch.py` or so instead, and also install that in the Dockerfile (e.g. via `pip install pytest`)

**Comment by RalfJung** [comment]
> I suppose we need to use Docker here so we have a systemd available, right?

Yes. Travis uses Ubuntu 16.04 which contains systemd 204 which is too old for the python module.

File: .travis.yml (hunk 1 above)
Comment (RalfJung) [reply .travis.yml]: > Why $(pwd) and not $PWD?

I did not know about `$PWD`. ;)

> you're running the Python file here, which does nothing.

Oh. Maybe the README should say that. This is also how I ran the tests before submitting the other PRs.^^

> I'd prefer to get the newest version via pip though.

Sure. Same for python3-systemd?

**Comment by The-Compiler** [comment]
LGTM now, and seems like the tests still pass 😆 Thanks!

**Comment by The-Compiler** [comment]
I now added a [development](https://github.com/The-Compiler/journalwatch#development) section to the readme.

**Pull request merged**